themes/theme-gmd/components/AppBar/presets/DefaultBar/index.jsx

```diff
@@ -71,6 +71,11 @@ class AppBarDefault extends PureComponent {
 
     if (this.props.route.visible) {
       this.updateStatusBar();
+
+      if (this.props.title) {
```

Comment:
This code is executed within the componentDidMount lifecycle hook. So will not run when the title is updated e.g. for "pages" or deeplinks where a request needs to be dispatched before we have the page title.

Reply:
True. It was forgotten in PageContent to pass a `Title` to Bar component